### PR TITLE
add request

### PR DESCRIPTION
**Pull request by quanghv96**

### CODE REVIEW

**Comment by quanghv96** [comment]
ready

File: app/Http/Requests/CustomerUpdateRequest.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required',
+            'phone' => 'required',
+            'email' => 'required|email|unique:users,email,'.$this->segment(3),
```
Comment (BunnyPi04) [inline app/Http/Requests/CustomerUpdateRequest.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by quanghv96** [comment]
ready

**Pull request merged**